src/docs/GetStarted/Designers/namingConvention/index.mdx

```diff
@@ -1,8 +1,40 @@
 ---
-title: "Naming convention"
+title: 'Naming convention'
 order: 5
 ---
 
-#About the naming convention
+> Mozaic is a product used by a lot of people. When so many people work together from different places on such a big project, you need to establish some ground rules that everyone needs to follow. These rules must be followed across every tool your team is using to create consistency and simplify the collaboration process.
 
-Xxxxxxxx
+## English for everyone
+
+Every file, layer, symbol and so on must be named in english.
+
+## File Naming Conventions
+
+The prefix always contains two letters depending on the pattern type and look like that: **_[XX] File name_**
+
+| Prefix   | Pattern type        | Example                                     |
+| -------- | ------------------- | ------------------------------------------- |
+| **[AA]** | Styles              | Colors, icons, fonts, shadows, ...          |
+| **[BA]** | Simple Components   | Buttons, links, tags, forms, ...            |
+| **[BB]** | Complex Components  | Notifications, accordions, popins, tabs ... |
+| **[ZZ]** | Specific librairies | Tools, documentation, ...                   |
```

Comment:
I would keep this in the contributing as well (as well as any relevant information), to avoid reader having to go up and down in the docs to get useful informations